docs/iris/src/developers_guide/code_format.rst

```diff
@@ -14,7 +14,10 @@ and then manually run from the root of the Iris repository::
   $ black .
   $ flake8 .
 
-Alternatively, we recommend using `pre-commit <https://pre-commit.com/>`_ to automatically run ``black`` and ``flake8``
+Alternatively, ``black`` and ``flake8`` may be installed using ``conda`` instead. Note that, ``black`` also offers
```

Comment:
Having these two sentances on the same line seems to suggest they are talking about related ideas. To me, it seems as though editor integration is more closely related to the idea of workflows below.